packages/react-menu/src/MenuList/MenuList.types.ts

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+import { ComponentProps } from '@fluentui/react-utils';
+
+export interface MenuListProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {}
+
+export interface MenuListState extends MenuListProps {}
```

Comment:
```suggestion
export interface MenuListState extends MenuListProps {
  ref: SMTH
}
```

`ref` should be also typed

Reply:
typed to the same output as `useMergedRefs`